tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -304,12 +304,15 @@ def pid(self):
 
     def on_output(self, line):
         """Write a line of output from the firefox process to the log"""
-        data = line.decode("utf8", "replace")
-        if self.stack_fixer:
-            data = self.stack_fixer(data)
-        self.logger.process_output(self.pid(),
-                                   data,
-                                   command=" ".join(self.runner.command))
+        if "GLib-GObject-CRITICAL" in line:
+            return
+        if(line):
```

Comment:
Tiny nit: this should be `if line:`. Otherwise, this looks good to me. (But I'd still like @jgraham to check.)